inc/Admin.php

```diff
@@ -27,8 +27,8 @@ public function __construct() {
 		/* Add inline style to hide subnav link */
 		\add_action( 'admin_head', array( __CLASS__, 'admin_nav_style' ) );
 
-		\add_filter('newfold-runtime', array( __CLASS__, 'add_to_runtime' ) );
-		\add_filter('newfold_runtime', array( __CLASS__, 'add_to_runtime' ) );
+		\add_filter( 'newfold-runtime', array( __CLASS__, 'add_to_runtime' ) );
```

Comment:
Do we still need both of these in place? I believe `newfold_runtime` replaced `newfold-runtime`.